content/en/cloudcraft/faq/how-cloudcraft-connects-to-aws.md

```diff
@@ -0,0 +1,20 @@
+---
+title: How does Cloudcraft connect to my AWS account?
+kind: guide
+---
+
+Cloudcraft makes use of cross-account roles to access your AWS account, which is considered [the secure way to access your AWS environment][1] by AWS themselves. Because of this, to use Cloudcraft connected with your AWS account, you need to create a secure read-only role in your AWS account that is specific to Cloudcraft and can easily be revoked at any time.
```

Comment:
```suggestion
Cloudcraft uses cross-account roles to access your AWS account, which is considered [the secure way to access your AWS environment][1] by AWS themselves. Because of this, you must create a secure read-only role in your AWS account that is specific to Cloudcraft and can easily be revoked at any time.
```